scripts/setup.sh

```diff
@@ -89,11 +83,8 @@ setup() {
   fi
 
   msg "Setting up GHC if needed..."
-  stack setup --verbosity warning
-  if [ $? -ne 0 ] ; then
-    err "Stack setup failed with error $?. Aborting..."
-    exit 1
-  fi
+  stack setup --verbosity warning ; RETCODE=$?
+  [ ${RETCODE} -ne 0 ] && exit_err "Stack setup failed with error ${RETCODE}. Aborting..."
```

Comment:
Yup, check https://github.com/begriffs/haskell-vim-now/files/47298/haskell_vim_now_2015-11-30.txt, last line. It aborts and installation (so $? in condition is not 0), then prints it to be 0.